crml/staking/src/multi_token_economy_tests.rs

```diff
@@ -274,14 +263,15 @@ pub fn start_era(era_index: EraIndex) {
 	assert_eq!(Staking::current_era(), era_index);
 }
 
-pub fn current_total_payout_for_duration(duration: u64) -> u64 {
-	inflation::compute_total_payout(
-		<Test as Trait>::RewardCurve::get(),
-		<Module<Test>>::slot_stake() * 2,
-		GenericAsset::total_issuance(&STAKING_ASSET_ID),
-		duration,
-	)
-	.0
+pallet_staking_reward_curve::build! {
```

Comment:
Wasn't the same curve from the mock, usable here?

Reply:
nice spotting, turns out it's not needed here at all